l10n_ec_ote/data/res.country.state.csv

```diff
@@ -0,0 +1,2 @@
+id,country_id:id,code,name
+base.state_ec_90,base.ec,90,ZONA NO DELIMITADA
```

Comment:
Because this record? this is not State than exist

Reply:
Some territories are not defined as belonging to a state or city 
https://es.wikipedia.org/wiki/Anexo:Cantones_de_Ecuador#Zonas_no_delimitadas